packages/gatsby-theme-aio/src/components/Search/index.js

```diff
@@ -174,6 +174,8 @@ const Search = ({ algolia, searchIndex, indexAll, showSearch, setShowSearch, sea
       setQueryStringParameter(SEARCH_PARAMS.keywords, selectedKeywords);
       setQueryStringParameter(SEARCH_PARAMS.index, selectedIndex);
 
+      isIFramed ? parent.postMessage(JSON.stringify({ 'query': searchQuery, 'keywords': selectedKeywords, 'index': selectedIndex }), window.location.hostname === 'localhost' ? 'http://localhost:3000' : window.location.hostname === 'developer.adobe.com' ? 'https://developer.adobe.com' : '*') : '';
+
```

Comment:
Following up on slack conversation with @timkim: 

- I will be refactoring the target origin selection into a helper function with if/else to improve readability
- I will look into grabbing parent origin from within the iframe to better handle helix stage origins and retire the wildcard `*`